settings.py

```diff
@@ -2015,6 +2020,9 @@ def _pkce_required(client_id):
 
 REST_FRAMEWORK = {
     'DATETIME_FORMAT': '%Y-%m-%dT%H:%M:%S.%fZ',
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+       'rest_framework.authentication.TokenAuthentication',
+   ),
```

Comment:
what all does this impact outside of the abdm app?

Reply:
This is removed.